packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -283,6 +284,62 @@ describe('StripeHelper', () => {
     });
   });
 
+  describe('createSetupIntent', () => {
+    it('creates a setup intent', async () => {
+      const expected = deepCopy(newSetupIntent);
+      sandbox
+        .stub(stripeHelper.stripe.setupIntents, 'create')
+        .returns(expected);
+
+      const actual = await stripeHelper.createSetupIntent('cust_new');
+
+      assert.deepEqual(actual, expected);
+      assert.hasAnyKeys(actual, 'client_secret');
+    });
+
+    it('surfaces stripe errors', async () => {
+      const apiError = new stripeError.StripeAPIError();
+      sandbox
+        .stub(stripeHelper.stripe.setupIntents, 'create')
+        .rejects(apiError);
+
+      return stripeHelper.createSetupIntent('cust_new').then(
+        () => Promise.reject(new Error('Method expected to reject')),
+        (err) => {
+          assert.equal(err, apiError);
+        }
+      );
+    });
+  });
+
+  describe('updateDefaultPaymentMethod', () => {
+    it('updates the default payment method', async () => {
+      const expected = deepCopy(newCustomerPM);
+      sandbox.stub(stripeHelper.stripe.customers, 'update').returns(expected);
```

Comment:
s/returns/resolves/ here and various other spots.  I can file a PR for only that change throughout this file and subscriptions route tests.

Reply:
Indeed, this does work, but only out of an oddly happy accident. A substantial amount of other tests also are using returns here, which is where I originally copied it from. They should all be updated for resolves. Separate PR for that seems good as it will update more tests than just these.